test/models/game_test.exs

```diff
@@ -1,18 +1,44 @@
 defmodule CoverMyPingPong.GameTest do
-  use CoverMyPingPong.ModelCase
+  use CoverMyPingPong.{ModelCase}
+  import CoverMyPingPong.Factory
 
-  alias CoverMyPingPong.Game
+  alias CoverMyPingPong.{Game}
 
-  @valid_attrs %{match_winner: "some content", opponent: "some content", opponent_score: 42, player: "some content", player_score: 42}
+  @valid_attrs %{
+    player_id: 1,
+    opponent_id: 2,
+    player_score: Enum.random(0..21),
+    opponent_score: Enum.random(0..21),
+    match_winner_id: 1,
+    played_at_date: Timex.today,
+    location: Enum.random(["Columbus", "Cleveland"])
+  }
   @invalid_attrs %{}
 
-  test "changeset with valid attributes" do
-    changeset = Game.changeset(%Game{}, @valid_attrs)
-    assert changeset.valid?
-  end
+  describe "changeset/2" do
+    test "with valid attributes is valid" do
+      changeset = Game.changeset(%Game{}, @valid_attrs)
+      assert changeset.valid?
+    end
+
+    test "is invalid without a player id" do
+      attrs = Map.delete(@valid_attrs, :player_id)
+      changeset = Game.changeset(%Game{}, attrs)
+      refute changeset.valid?
+    end
+
+    test "is invalid without an opponent id" do
```

Comment:
Yeah, forgot to. I was too excited to see my tests finally pass with Travis that I just wanted to get it pushed